src/pages/payment-methods/_payment-data.js

```diff
@@ -417,55 +447,163 @@ const payment_data = [
                 withdrawal_time: <Localize translate_text="Subject to internal checks" />,
                 description: (
                     <Localize translate_text="Deposit and withdraw in Litecoin, a durable cryptocurrency with fast transaction processing." />
                 ),
-                name: 'litecoin',
+                name: 'LTC',
                 reference: 'litecoin-payment-method.pdf',
             },
+            {
+                method: <PaxosIc />,
+                currencies: 'PAX',
+                min_max_deposit: <Localize translate_text="No minimum" />,
+                min_max_withdrawal: undefined,
+                deposit_time: <Localize translate_text="Funds available as soon as confirmed" />,
+                withdrawal_time: <Localize translate_text="Subject to internal checks" />,
+                description: (
+                    <Localize translate_text="Deposit and withdraw in Paxos Standard, a cryptocurrency that’s pegged to USD." />
+                ),
+                name: 'PAX',
+            },
+            {
+                method: <TrueIc />,
+                currencies: 'TUSD',
+                min_max_deposit: <Localize translate_text="No minimum" />,
+                min_max_withdrawal: undefined,
+                deposit_time: <Localize translate_text="Funds available as soon as confirmed" />,
+                withdrawal_time: <Localize translate_text="Subject to internal checks" />,
+                description: (
+                    <Localize translate_text="Deposit and withdraw in TrueUSD, a cryptocurrency that’s pegged to USD." />
+                ),
+                name: 'TUSD',
+            },
+            {
+                method: <UsdCoinIc />,
+                currencies: 'USDC',
+                min_max_deposit: <Localize translate_text="No minimum" />,
+                min_max_withdrawal: undefined,
+                deposit_time: <Localize translate_text="Funds available as soon as confirmed" />,
+                withdrawal_time: <Localize translate_text="Subject to internal checks" />,
+                description: (
+                    <Localize translate_text="Deposit and withdraw in USD Coin, a cryptocurrency that’s pegged to USD." />
+                ),
+                name: 'USDC',
+            },
+            {
+                method: <UsdkIc />,
+                currencies: 'USDK',
+                min_max_deposit: <Localize translate_text="No minimum" />,
+                min_max_withdrawal: undefined,
+                deposit_time: <Localize translate_text="Funds available as soon as confirmed" />,
+                withdrawal_time: <Localize translate_text="Subject to internal checks" />,
+                description: (
+                    <Localize translate_text="Deposit and withdraw in USDK, a cryptocurrency that’s pegged to USD." />
+                ),
+                name: 'USDK',
+            },
+            {
+                method: <TetherIc />,
```

Comment:
is the icon for usdt and eusdt is thether?

Reply:
it's based on the design